packages/data/src/world/mm/great_bay_temple.yml

```diff
@@ -52,7 +52,7 @@
   events:
     MAGIC: "true"
   exits:
-    "Great Bay Temple Baba Room": "has_mask_zora || ((has_bombchu || has_mask_blast) && (short_hook_anywhere || underwater_walking)) || hookshot_anywhere"
+    "Great Bay Temple Baba Room": "has_mask_zora || ((has_bombchu || has_mask_blast) && (short_hook_anywhere || underwater_walking || can_dive_big)) || hookshot_anywhere"
```

Comment:
This will require zora tunic as it takes too much time to reach the room left as is. If it is extended to a 300 frame delay on the timer, then it would be fine, but i'd still say it should at least soft require the tunic.